python/ray/train/lightning/lightning_utils.py

```diff
@@ -49,6 +50,36 @@ def get_worker_root_device():
         return devices
 
 
+@PublicAPI(stability="alpha")
+def get_devices() -> Optional[Union[List[int], str]]:
+    """Returns the parallel devices for Lightning Trainer on each Ray Train worker.
+
+    This method returns the list of CUDA device indexes of a GPU worker, and returns 
+    "auto" if called inside a CPU worker. Note that you can only call this method within 
+    the training function of the :class:`TorchTrainer <ray.train.torch.TorchTrainer>` class.
+
+    Example:
+        .. testcode::
+
+            import ray
+            import pytorch_lightning as pl
```

Comment:
nit: Should we just `import lightning` instead?

Reply:
I am also thinking of updating all our import path from `import pytorch_lightning` to `import lightning.pytorch`. But this will drop backward compatibility for all lighting 1.x users. 

Maybe 2.7 it's the time do so. I suggest to make a separate PR to update the import path, since it involves more code changes in all unittest, release test and examples.